db/schema.sql

```diff
@@ -265,6 +265,7 @@ CREATE TABLE `future` (
   KEY `event_date` (`event_date`),
   KEY `modified` (`modified`)
 );
+ALTER TABLE `future` CONVERT TO CHARACTER SET 'utf8';
```

Comment:
I think you can do this in the create table statement with `) CHARACTER SET 'utf8'`?